src/thor/trace_attributes_action.cc

```diff
@@ -35,35 +35,6 @@ constexpr size_t kTripLegIndex = 3;
 
 namespace valhalla {
 namespace thor {
-
-void thor_worker_t::filter_attributes(const valhalla_request_t& request,
```

Comment:
technically this definition didn't need to move as its in a header that is shared by all the other actions